crates/bevy_reflect/src/func/reflect_fn.rs

```diff
@@ -76,8 +76,9 @@ pub trait ReflectFn<'env, Marker>: ReflectFnMut<'env, Marker> {
 /// - `Fn(&mut Receiver, arg0, arg1, ..., argN) -> &mut R`
 /// - `Fn(&mut Receiver, arg0, arg1, ..., argN) -> &R`
 macro_rules! impl_reflect_fn {
-    ($(($Arg:ident, $arg:ident)),*) => {
```

Comment:
I noticed that this didn't show up in the docs and found out that it's behind the `functions` feature. 
So I rebuilt with `--all-features` and got a `#[doc(fake_variadic)] must be used on the first of a set of tuple or fn pointer trait impls with varying arity` error :/

Reply:
I'll fix that, thanks.